tests/modules/ejector/test_ejector.py

```diff
@@ -0,0 +1,460 @@
+        ejector._get_sweep_delay_in_epochs = Mock(return_value=1)
+        ejector._get_total_balance = Mock(return_value=50)
+        ejector.validators_state_service.get_recently_requested_but_not_exited_validators = Mock(
+            return_value=[]
+        )
+
+        result = ejector.get_validators_to_eject(ref_blockstamp)
+        assert result == [], "Unexpected validators to eject"
+
+    # NOTE: not sure if this test makes sense
+    # @pytest.mark.unit
+    # @pytest.mark.usefixtures("consensus_client")
+    # def test_simple(
+    #     self,
+    #     ejector: Ejector,
+    #     ref_blockstamp: ReferenceBlockStamp,
+    #     validator_to_exit_it: ValidatorToExitIterator,
+    #     chain_config: ChainConfig,
+    # ):
+    #     def _lido_validator(index: int):
+    #         return Mock(index=index, validator=Mock(pubkey="0x00"))
+    #
+    #     validators = [
+    #         ((StakingModuleId(0), NodeOperatorId(1)), _lido_validator(0)),
+    #         ((StakingModuleId(0), NodeOperatorId(3)), _lido_validator(1)),
+    #         ((StakingModuleId(0), NodeOperatorId(5)), _lido_validator(2)),
+    #     ]
+    #
+    #     validator_to_exit_it.__iter__ = Mock(return_value=iter(validators))
+    #
+    #     ejector.get_chain_config = Mock(return_value=chain_config)
+    #     ejector.get_total_unfinalized_withdrawal_requests_amount = Mock(
+    #         return_value=200
+    #     )
+    #     ejector.prediction_service.get_rewards_per_epoch = Mock(return_value=1)
+    #     ejector._get_sweep_delay_in_epochs = Mock(return_value=ref_blockstamp.ref_epoch)
+    #     ejector._get_total_balance = Mock(return_value=100)
+    #     ejector.validators_state_service.get_recently_requested_but_not_exited_validators = Mock(
+    #         return_value=[]
+    #     )
+    #
+    #     ejector._get_withdrawable_lido_validators = Mock(return_value=0)
+    #     ejector._get_predicted_withdrawable_epoch = Mock(return_value=50)
+    #     ejector._get_predicted_withdrawable_balance = Mock(return_value=50)
+    #
+    #     result = ejector.get_validators_to_eject(ref_blockstamp)
+    #     assert result == [validators[0]], "Unexpected validators to eject"
+
+
+@pytest.mark.unit
+@pytest.mark.usefixtures("contracts")
+def test_get_unfinalized_steth(ejector: Ejector, blockstamp: BlockStamp) -> None:
+    result = ejector.get_total_unfinalized_withdrawal_requests_amount(blockstamp)
+    assert result == 2181000000000000000, "Unexpected unfinalized stETH"
+
+
+@pytest.mark.unit
+def test_compute_activation_exit_epoch(
+    ejector: Ejector,
+    ref_blockstamp: ReferenceBlockStamp,
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    with monkeypatch.context() as m:
+        m.setattr(ejector_module, "MAX_SEED_LOOKAHEAD", 17)
+        result = ejector.compute_activation_exit_epoch(ref_blockstamp)
+        assert result == 3546 + 17 + 1, "Unexpected activation exit epoch"
```

Comment:
Set `ref_epoch` for blockstamp explicitly

Reply:
and in other tests too if it needed

Reply:
Defined at fixture level.

Reply:
Not defined at this moment. I think we shouldn't expect constant values from global-scoped objects. If the test requires the number, it should be passed here

Reply:
Moved to the test body.